exchanges/deribit/deribit_wrapper.go

```diff
@@ -331,15 +338,22 @@ func (d *Deribit) UpdateOrderbook(ctx context.Context, p currency.Pair, assetTyp
 		Asset:           assetType,
 		VerifyOrderbook: d.CanVerifyOrderbook,
 	}
-	fmtPair, err := d.FormatExchangeCurrency(p, assetType)
-	if err != nil {
-		return nil, err
+	var instrumentID string
+	var err error
+	if assetType == asset.Futures {
+		instrumentID = d.formatFuturesTradablePair(p)
+	} else {
+		p, err = d.FormatExchangeCurrency(p, assetType)
+		if err != nil {
+			return nil, err
+		}
+		instrumentID = p.String()
 	}
 	var obData *Orderbook
 	if d.Websocket.IsConnected() {
-		obData, err = d.WSRetrieveOrderbookData(fmtPair.String(), 50)
+		obData, err = d.WSRetrieveOrderbookData(instrumentID, 50)
 	} else {
-		obData, err = d.GetOrderbookData(ctx, fmtPair.String(), 50)
+		obData, err = d.GetOrderbookData(ctx, instrumentID, 50)
```

Comment:
[ERROR] | ORDERBOOK | 05/06/2023 09:37:55 | Failed to get REST orderbook for Deribit BTC-FS-30JUN23_PERP future_combo. Error: cannot load book for exchange Deribit pair BTC-FS-30JUN23_PERP asset future_combo for Bids: pricing out of order